packages/scripts/README.md

```diff
@@ -350,6 +351,26 @@ Jest will look for test files with any of the following popular naming conventio
 
 It uses [Jest](https://jestjs.io/) behind the scenes and you are able to use all of its [CLI options](https://jestjs.io/docs/en/cli.html). You can also run `./node_modules/.bin/wp-scripts test:unit --help` or `npm run test:unit:help` (as mentioned above) to view all of the available options. By default, it uses the set of recommended options defined in [@wordpress/jest-preset-default](https://www.npmjs.com/package/@wordpress/jest-preset-default) npm package. You can override them with your own options as described in [Jest documentation](https://jestjs.io/docs/en/configuration). Learn more in the [Advanced Usage](#advanced-usage) section.
 
+### `changelog`
+
+Based on a milestone, it creates a changelog entry with all PRs merged in that milestone.
+
```

Comment:
Should clarify this looks for a milestone with the title being _exactly_ a version number as this is what the script matches.  It might be better if we broaden the script matching (see my additional comment later) where we match on the title _containing_ the given version number.